teamcity.build

```diff
@@ -136,8 +124,6 @@
 				<include name="*.dll"/>
 			</fileset>
 		</copy>
```

Comment:
@fredericDelaporte shall we remove the oracle step (and binaries) completely? It wasn't build by TeamCity for more than a year.

Reply:
Non-built on TeamCity Oracle targets are: `oracle`, `oracle32`, `oracle64` and `oracle-managed64`. Only `oracle-managed32` is built.

Why not removing the unused targets. (`oracle32` and `oracle-managed64` are available as TeamCity built options in `ShowBuildMenu.bat`, and would have to be removed from there too, if removed from `teamcity.build`.)

About the binaries, they also serve for the `E. Add a test configuration for Oracle.` option of `ShowBuildMenu.bat`. So either this option is kept for people wishing to test with the unmanaged driver and then the binaries are to stay, or this option has to be removed too.